target/linux/ipq806x/base-files/etc/init.d/cpufreq

```diff
@@ -14,4 +14,12 @@ boot() {
     echo 10 > /sys/devices/system/cpu/cpufreq/ondemand/sampling_down_factor
     echo 50 > /sys/devices/system/cpu/cpufreq/ondemand/up_threshold
   fi
+
+  # Restore original implementation where the eth0 was used for the lan port
+  # and the eth2 was used for the wan port
```

Comment:
@ansuel
Nitpicking, but the comment "eth2" does not match the commands below "eth1", and the commit message uses still different terminology.

Maybe it might be "eth0/gmac1" and "eth1/gmac2" ?

Reply:
I also think the script name should be changed

Reply:
You might divide the script into two, as the cpufreq part is quite different thing.

Reply:
I split the script.